Makefile

```diff
@@ -55,3 +46,13 @@ verify:
 clean:
 	@GOOS=linux go clean -i -r -x ./...
 	-rm -rf _output
+	-rm -rf $(RELEASE_DIR)
+	-rm -rf ./linode-blockstorage-csi-driver
+
+.PHONY:
+release:
+	mkdir $(RELEASE_DIR)
```

Comment:
`mkdir -p` to avoid exiting if it exists... or maybe remove the dir before this? Should only be an issue when invoked locally since CI should have a clean env.